doc/howto/images_manage.md

```diff
@@ -1,20 +1,105 @@
+You can also export an image to a file, which can be useful to {ref}`copy or import it <images-copy>` on another machine.
 
+## List available images
 
+To list all images on a server, enter the following command:
 
-## Export images
-Use:
+    lxc image list [<remote>:]
 
-	lxc image export imagename [target folder] [flags]
+If you do not specify a remote, the {ref}`default remote <images-remote-default>` is used.
 
-Flags:
+(images-manage-filter)=
+### Filter available images
 
-`--vm` - Query virtual machine images
+To filter the results that are displayed, specify a part of the alias or fingerprint after the command.
+For example, to show all Debian images, enter the following command:
+
+    lxc image list images: debian
+
+You can specify several filters as well.
+For example, to show all 64-bit Debian images, enter the following command:
+
+    lxc image list images: debian amd64
+
+To filter for properties other than alias or fingerprint, specify the filter in `<key>=<value>` format.
+For example:
+
+    lxc image list images: debian architecture=x86_64
+
+## View image information
+
+To view information about an image, enter the following command:
+
+    lxc image info <image_ID>
+
+As the image ID, you can specify either the image's alias or its fingerprint.
+For a remote image, remember to include the remote server (for example, `images:ubuntu/22.04`).
+
+To display only the image properties, enter the following command:
+
+    lxc image show <image_ID>
+
+You can also display a specific image property (located under the `properties` key) with the following command:
+
+    lxc image get-property <image_ID> <key>
+
+For example, to show the release name of the official Ubuntu 22.04 image, enter the following command:
+
+    lxc image get-property ubuntu:22.04 release
+
+(images-manage-edit)=
+## Edit image properties
+
+To set a specific image property that is located under the `properties` key, enter the following command:
+
+    lxc image set-property <image_ID> <key>
+
+To edit the full image properties, including the top-level properties, enter the following command:
+
+    lxc image edit <image_ID>
+
+## Configure image aliases
+
+Configuring an alias for an image can be useful to make it easier to refer to an image, since remembering an alias is usually easier than remembering a fingerprint.
+Most importantly, however, you can change an alias to point to a different image, which allows creating an alias that always provides a current image (for example, the latest version of a release).
+
+You can see some of the existing aliases in the image list.
+To see the full list, enter the following command:
+
+    lxc image alias list
+
+You can directly assign an alias to an image when you {ref}`copy or import <images-copy>` or {ref}`publish <images-create-publish>` it.
+Alternatively, enter the following command:
+
+    lxc image alias create <alias> <image_fingerprint>
```

Comment:
We should probably use `<alias_name>` here to align with other examples, such as https://linuxcontainers.org/lxd/docs/master/howto/instances_create/#usage where it uses `<image_name> <instance_name>`.